airbyte-integrations/connectors/source-stripe/source_stripe/source.py

```diff
@@ -109,7 +108,7 @@ def check_connection(self, logger: logging.Logger, config: MutableMapping[str, A
         args = self._get_stream_base_args(config)
         account_stream = StripeStream(name="accounts", path="accounts", use_cache=USE_CACHE, **args)
         try:
-            next(account_stream.read_records(sync_mode=SyncMode.full_refresh))
+            next(account_stream.read_records(sync_mode=SyncMode.full_refresh), None)
```

Comment:
This fixes:

- https://github.com/orgs/airbytehq/projects/23/views/17?pane=issue&itemId=79303359